pilot/pkg/xds/ep_filters.go

```diff
@@ -133,6 +135,37 @@ func (b *EndpointBuilder) EndpointsByNetworkFilter(endpoints []*LocLbEndpointsAn
 	return filtered
 }
 
+func (b *EndpointBuilder) mTLSDisabled(lbEp *endpoint.LbEndpoint) bool {
+	// TODO share code with cluster builder. if the cluster does not have the transportSocket(Match), mTLS is disabled
+	if b.push.Mesh != nil && b.push.Mesh.EnableAutoMtls != nil && !b.push.Mesh.EnableAutoMtls.Value {
+		return true
```

Comment:
so if auto mtls is disabled, then we cannot do cross-network?

Reply:
no – I don't remember why I added this. Will remove.